src/core/main.c

```diff
@@ -83,6 +84,7 @@ int main(int argc, char *argv[]) {
 		logprint(ERROR, "wayland: failed to connect to display");
 		goto error;
 	}
+	logprint(INFO, "wlroots: wl_display fd: %d", wl_display_get_fd(wl_display));
```

Comment:
Hmm, is the FD number useful?

Reply:
No, I was probably debugging something long before I added decent logging, and mindlessly s/printf(/logprint(INFO,/